perfkitbenchmarker/disk.py

```diff
@@ -63,6 +63,9 @@ def __init__(self, disk_spec):
     self.mount_point = disk_spec.mount_point
     self.iops = disk_spec.iops
 
+    # Windows related attributes.
```

Comment:
This isn't necessarily Windows specific. I think keeping count of the per-instance disks would benefit some Linux implementations also, such as the rather ugly /dev/vdN name allocation in azure_disk.py. If the base class can track this generically, would be nice to document this.